association.go

```diff
@@ -200,11 +220,20 @@ func createAssocation(netConn net.Conn) *Association {
 		handshakeCompletedCh:    make(chan error),
 		cumulativeTSNAckPoint:   tsn - 1,
 		advancedPeerTSNAckPoint: tsn - 1,
+		silentError:             errors.New("sliently discard"),
+		log:                     config.LoggerFactory.NewLogger("sctp"),
 	}
 
+	// RFC 4690 Sec 7.2.1
+	//  o  The initial cwnd before DATA transmission or after a sufficiently
+	//     long idle period MUST be set to min(4*MTU, max (2*MTU, 4380
+	//     bytes)).
+	a.cwnd = min32(4*a.mtu, max32(2*a.mtu, 4380))
+	a.log.Tracef("updated cwnd=%d ssthresh=%d inflight=%d (INI)", a.cwnd, a.ssthresh, a.inflightQueue.getNumBytes())
+
 	a.t1Init = newRTXTimer(timerT1Init, a, maxInitRetrans)
 	a.t1Cookie = newRTXTimer(timerT1Cookie, a, maxInitRetrans)
-	a.t3RTX = newRTXTimer(timerT3RTX, a, pathMaxRetrans)
+	a.t3RTX = newRTXTimer(timerT3RTX, a, 0 /* rtx forever */)
```

Comment:
Perhaps 0 is a constant rather than a comment? Not 100% sure on this one

Reply:
Good idea.

Reply:
Fixed at ef7368e